waspc/src/Wasp/AI/GenerateNewProject/PageComponentFile.hs

```diff
@@ -63,7 +63,6 @@ fixPageComponent newProjectDetails waspFilePath pageComponentPath = do
             - Make sure to use only queries and actions that are defined in the Wasp file (listed below)!
             - Ensure query and action imports are correct. One import per query / action, default imports,
               name of the file same as name of the query.
-            - Don't use `useAction` or `useMutation`! Use actions directly.
```

Comment:
I would not just delete it but replace it with this:
```
 - Don't use `useMutation`, instead use `useAction`! Example of usage: `const createTaskFn = useAction(createTask);`.
```

Reply:
I don't think it's necessary. We already told it about the `useAction` hook the same way we told it about the `useQuery` hook. These bullet points don't mention `useQuery` (or `useAction`) and they seem to work well.